src/App.js

```diff
@@ -4,9 +4,22 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  handleQueryChange = (event) => {
+    this.setState({ query: event.target.value });
```

Comment:
```suggestion
    this.setState({ query: event.target.value.toLowerCase() });
```
Это контролируемый ввод, можешь сразу тут приводить

Reply:
good advice